### PR TITLE
add solutions for ticket 4

### PR DESCRIPTION
**Pull request by PabloJulianRial**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -43,3 +43,28 @@ describe("all non existing endpoint", () => {
       });
   });
 });
+describe("GET /api/articles/:article_id", () => {
+  test("status 200: responds with the queried article", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({body}) =>{
+        expect(body.article).toHaveProperty("title", expect.any(String));
```
Comment (pippaamy) [inline __tests__/app.test.js:53]: we should expect it to have an article_id of 1

File: __tests__/app.test.js [hunk 2]
```diff
@@ -43,3 +43,28 @@ describe("all non existing endpoint", () => {
...
+      })
+      
+  });
+  test('status 400: responds with an error message when an invalid parameter is used', () =>{
+    return request(app)
+    .get('/api/articles/banana')
+    .expect(400)
+    .then(({body}) =>{
+      expect(body.message).toBe('Bad request')
+    })
```
Comment (pippaamy) [inline __tests__/app.test.js]: what if the article id is 900000

File: controllers/ncnews.controllers.js [hunk 3]
```diff
@@ -11,3 +12,11 @@ exports.getTopics = (req, res, next) => {
 exports.getEndpoints = (req, res) => {
   res.status(200).send(endpointsData);
 };
+exports.getArticleById = (req, res, next) => {
+  const {article_id} = req.params //id or article_id????
```
Comment (pippaamy) [inline controllers/ncnews.controllers.js]: remember to get rid of comments

**Review by pippaamy — COMMENTED** [review]
good job, some changes to make to your tests :)

**Review by lewispricey — APPROVED** [review]
Great work on this ticket Pablo, thanks for implementing Pippa's feedback.

Good to merge when you're ready 🚀

**Pull request merged**